components/core/tools/scripts/db/init-db.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env python3
```

Comment:
@kirkrodrigues is this script still used?
I feel I haven't seen anyone updated this script for a while, wonder should we check if the content of this script is up-to-date with all latest feature. or maybe we can deprecate this script?